src/workers/personalizedDigestEmail.ts

```diff
@@ -213,18 +218,34 @@ const digestTypeToFunctionMap: Record<
       },
     );
   },
-  [UserPersonalizedDigestType.Brief]: async () => {
-    // brief is sent through different workers after generation
+  [UserPersonalizedDigestType.Brief]: async (data, con, logger) => {
+    const { userId } = data.personalizedDigest;
+    const postId = await generateShortId();
+
+    const post = con.getRepository(BriefPost).create({
+      id: postId,
+      shortId: postId,
+      authorId: userId,
+      private: true,
+      visible: false,
+    });
+
+    await con.getRepository(BriefPost).save(post);
+
+    triggerTypedEvent(logger, 'api.v1.brief-generate', {
+      payload: new UserBriefingRequest({
+        userId,
+        frequency: data.personalizedDigest.flags.sendType,
+        modelName: BriefingModel.Default,
+      }),
+      postId,
+    });
   },
 };
 
 const worker: Worker = workerToExperimentWorker({
   subscription: 'api.personalized-digest-email',
   handler: async (message, con, logger, pubsub, allocationClient) => {
-    if (process.env.NODE_ENV === 'development') {
-      return;
-    }
-
```

Comment:
Was not really needed since we have `enablePersonalizedDigest` env as well which disables the workers locally by defualt